src/main/scala/org/opennetworkinsight/OniLDACWrapper.scala

```diff
@@ -17,24 +17,26 @@ import scala.sys.process._
 
 object OniLDACWrapper {
 
-  def runLDA(docWordCount: RDD[String], modelFile: String, topicDocumentFile: String, topicWordFile: String,
+  case class OniLDACInput(doc: String, word: String, count: Int) extends Serializable
+
+
+  def runLDA(docWordCount: RDD[OniLDACInput], modelFile: String, topicDocumentFile: String, topicWordFile: String,
              mpiPreparationCmd: String, mpiCmd: String, mpiProcessCount: String, mpiTopicCount: String,
              localPath: String, ldaPath: String, localUser: String, dataSource: String, nodes: String):
   (Array[String], Array[String])
   =
   {
-    val documentWordData = docWordCount.map(_.split(","))
     // Create word Map Word,Index for further usage
     val wordDictionary: Map[String, Int] = {
```

Comment:
yeah, just make a note of it and we can get it another pass ...

a lot of the code needs some variable rename and style passes... those should be easy runs but let's not forget anything